CODEOWNERS

```
@@ -233,7 +233,7 @@ esphome/components/ina2xx_base/* @latonita
 esphome/components/ina2xx_i2c/* @latonita
 esphome/components/ina2xx_spi/* @latonita
 esphome/components/inkbird_ibsth1_mini/* @fkirill
-esphome/components/inkplate6/* @jesserockz
+esphome/components/inkplate/* @jesserockz @JosipKuci
```

Weird alphabetical rules puts upper in front of lower :see_no_evil: 
```suggestion
esphome/components/inkplate/* @JosipKuci @jesserockz
```